### ISSUE TITLE
Weird Facebook mapping

### ISSUE THREAD

**Issue by sergeyf** [allenai/S2AFF#10]
Facebook AI Research
Facebook
Facebook AI Research (FAIR)
Meta Platforms, Inc.
Facebook inc
Facebook AI Research Paris

Are all mapped to 

https://ror.org/02388em19

Instead of https://ror.org/020ye1821 or https://ror.org/01zbnvs85

**Comment by sergeyf** [comment]
This is the fault of me relying on works_count to come from OpenAlex, which got it wrong:

```
Meta (Israel) 5020
Meta (United Kingdom) 73
Meta (United States) 1411
```

**Comment by sergeyf** [comment]
There are 542 pairs where the parent ROR has fewer citations than the child ROR and they have names like

NAME (location_1) and NAME (location_2)

Solution: swap the works_count between parents and kids that have this structure? It'll privilege the parents, which makes sense in most cases where only works_count is a tie-breaker...

**Comment by sergeyf** [comment]
Implemented solution as described above in #11
